resources/api-spec/hmi-api-health.json

```diff
@@ -0,0 +1,34 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "hmi-apim-health-api",
+    "version": "1.0"
+  },
+  "host": "hmi-apim.dev.platform.hmcts.net",
```

Comment:
Should this host be configurable / it replaces the dev with the relevant env it's running in?